packages/replay-next/src/suspense/HitPointsCache.ts

```diff
@@ -66,7 +66,10 @@ export const hitPointsCache = createFocusIntervalCacheForExecutionPoints<
         { kind: "locations", locations },
         { begin, end }
       );
-      hitPoints = pointDescriptions.map(({ point, time }) => ({ point, time }));
+      // Note that PointDescriptions include the current frame.
+      // Type-wise we're going to return `TimeStampedPoint`s,
+      // but the extra frame data is there in this case.
+      hitPoints = pointDescriptions;
```

Comment:
This should update the frame data like [here](https://github.com/replayio/devtools/blob/main/packages/replay-next/src/suspense/PauseCache.ts#L105-L112), otherwise we will get bugs due to corresponding sources.
When the frame data is updated, it's also safe to make this "official" by changing the return type from `TimeStampedPoint` to `PointDescription`.

Reply:
Yeah, I can definitely update the frame data, although the issue is that the other half of this cache _just_ gets `TimeStampedPoints`, I think?

Reply:
Ah, looks like those _are_ `PointDescription`s too.